controllers/auth.js

```diff
@@ -1,7 +1,11 @@
 const passport = require("passport");
 const users = require("../models/users");
+const QrCodeAuthModel = require("../models/qrCodeAuth");
 const authService = require("../services/authService");
-
+const { SOMETHING_WENT_WRONG } = require("../constants/errorMessages");
+const DATA_ADDED_SUCCESSFULLY = "User Device Info added successfully!";
```

Comment:
Would it be better to have separate file to save all constant messages
Need not be part of this PR
talking in general

Reply:
yes makes sense , i will create an issue ticket if required

Reply:
i moved them to constants now